src/daemon.rs

```diff
@@ -21,14 +22,29 @@ fn index() -> impl Responder {
 Use it with /validate?url=https.//.../gtfs.zip"#
 }
 
+#[post("/validate")]
+fn validate_post(body: web::Payload) -> impl Future<Item = Json<Response>, Error = Error> {
```

Comment:
shouldn't you also take the `Query<Params>` ?
this would remove the hardcoded `1000`